Preced_const_list:
          H_1: R_1

          H_2: R_2

          R_2: H_2

          R_3: H_3



 
  
  
